src/rgw/rgw_process.cc

```diff
@@ -395,6 +395,8 @@ int process_request(rgw::sal::Store* const store,
 
     ret = rgw_process_authenticated(handler, op, req, s, yield, store);
     if (ret < 0) {
+      if(ret == -EACCES)
+        s->err.message = "Access Denied";
```

Comment:
There isn't a mapping of generic error code vs error message, where can add this ?

Reply:
Added this error message in verify function as here it will overwrite other error message for -EACCES.